editor/src/components/inspector/inspector.tsx

```diff
@@ -165,9 +165,6 @@ const AlignmentButtons = React.memo((props: { numberOfTargets: number }) => {
       style={{
         justifyContent: 'space-around',
         height: UtopiaTheme.layout.rowHeight.normal,
-        position: 'sticky',
```

Comment:
wasn't this _meant_ to not make them scroll?

Reply:
that... would make sense. Do we want them not to scroll then?